sys/auto_init/auto_init.c

```diff
@@ -152,6 +156,10 @@ void auto_init(void)
     DEBUG("Auto init UDP module.\n");
     gnrc_udp_init();
 #endif
+#ifdef MODULE_GNRC_TCP
+    DEBUG("Auto init TCP module\n");
+    gnrc_tcp_init();
```

Comment:
In the entire auto_init.c nobody cares about return values. Why should I (I know its a lame excuse)?